### PR TITLE
file not doc

### PR DESCRIPTION
**Pull request by leohemsted**
## Checklist

- [x] I’ve used the pull request template
- [ ] I’ve written unit tests for these changes
- [x] I’ve updated the documentation in
  - `DOCUMENTATION.md`
  - `CHANGELOG.md`
- [x] I’ve bumped the version number in
  - `package.json`
- [ ] I've added new environment variables in
  - `CONTRIBUTING.md`
  - `notifications-node-client/scripts/generate_docker_env.sh`

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -295,11 +295,11 @@ The links are unique and unguessable. GOV.UK Notify cannot access or decrypt you
 1. Select __Edit__.
 1. Add a placeholder field to the email template using double brackets. For example:
 
-"Download your file at: ((link_to_document))"
+"Download your file at: ((link_to_file))"
 
 ### Upload your file
 
-The file you upload must be a PDF or CSV file smaller than 2MB. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
+The file you upload must be a PDF, CSV, text file or Microsoft Word document smaller than 2MB.. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
```
Comment (tombye) [inline DOCUMENTATION.md]: Needs the extra full-stop removing.

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -353,15 +353,15 @@ If the request is not successful, the promise fails with an `err`.
...
 |`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Service is not allowed to send documents"`<br>`}]`|Contact the GOV.UK Notify team|
 |`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error: Your system clock must be accurate to within 30 seconds"`<br>`}]`|Check your system clock|
 |`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Use the correct type of [API key](#api-keys)|
 |`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM/TEST/LIVE of 3000 requests per 60 seconds"`<br>`}]`|Refer to [API rate limits](#api-rate-limits) for more information|
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](#service-limits) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
-|`N\A`|`Document is larger than 2MB`|`Document is larger than 2MB`|
+|`N/A`|`File is larger than 2MB`|`The file is too big. Files must be smaller than 2MB`|
```
Comment (tombye) [inline DOCUMENTATION.md]: The 'How to fix' column value here is different to those in the other client PRs for this change.

File: DOCUMENTATION.md [hunk 3]
```diff
@@ -353,15 +353,15 @@ If the request is not successful, the promise fails with an `err`.
 |:---|:---|:---|
 |`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can't send to this recipient using a team-only API key"`<br>`]}`|Use the correct type of [API key](#api-keys)|
 |`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can't send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|Your service cannot send this notification in [trial mode](https://www.notifications.service.gov.uk/features/using-notify#trial-mode)|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Unsupported document type '{}'. Supported types are: {}"`<br>`}]`|The document you upload must be a PDF file|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Document didn't pass the virus scan"`<br>`}]`|The document you upload must be virus free|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Unsupported file type '(FILE TYPE)'. Supported types are: '(ALLOWED TYPES)'"`<br>`}]`|Wrong file type. You can only upload .pdf, .csv, .txt, .doc or .docx files|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "File did not pass the virus scan"`<br>`}]`|The file contains a virus|
```
Comment (tombye) [inline DOCUMENTATION.md]: The cell for the 'how to fix' column has changed from being an instruction (the payload needs to be ...) to a statement (the payload is ...). The [guidance on error messages](https://design-system.service.gov.uk/components/error-message/) seems to say we should stick to the first type.

Comment (tombye) [reply DOCUMENTATION.md]: Resolved for the same reasons as in the PHP client docs: https://github.com/alphagov/notifications-php-client/pull/69/files#r368542048

File: DOCUMENTATION.md (hunk 2 above)
Comment (leohemsted) [reply DOCUMENTATION.md]: This is the correct one - see https://docs.google.com/document/d/1QY_uot8rWFA5rF6VPFaFHxnLWnFCMCB2sSgUVv9IrpM/edit#

i've updated the other PRs to reflect this 
image

File: DOCUMENTATION.md [hunk 4]
```diff
@@ -295,11 +295,15 @@ The links are unique and unguessable. GOV.UK Notify cannot access or decrypt you
...
-"Download your file at: ((link_to_document))"
+"Download your file at: ((link_to_file))"
 
 ### Upload your file
 
-The file you upload must be a PDF or CSV file smaller than 2MB. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
+<<<<<<< HEAD
+The file you upload must be a PDF, CSV, text file or Microsoft Word document smaller than 2MB. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
+=======
+The file you upload must be a PDF, CSV, text file or Microsoft Word document smaller than 2MB.. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
```
Comment (tombye) [inline DOCUMENTATION.md]:
```suggestion
The file you upload must be a PDF, CSV, text file or Microsoft Word document smaller than 2MB. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) if you need to send other file types.
```

File: DOCUMENTATION.md [hunk 5]
```diff
@@ -832,7 +832,7 @@ If the request is not successful, the client will return an `HTTPError` containi
 |:---|:---|:---|
 |`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "id is not a valid UUID"`<br>`}]`|Check the notification ID|
 |`400`|`[{`<br>`"error": "PDFNotReadyError",`<br>`"message": "PDF not available yet, try again later"`<br>`}]`|Wait for the notification to finish processing. This usually takes a few seconds|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Document did not pass the virus scan"`<br>`}]`|You cannot retrieve the contents of a letter notification that contains a virus|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "File did not pass the virus scan"`<br>`}]`|You cannot retrieve the contents of a letter notification that contains a virus|
```
Comment (leohemsted) [inline DOCUMENTATION.md]: this error message (both for letters and doc dl) was changed in https://github.com/alphagov/notifications-api/pull/2696/commits/a16f79896b9cdc2ec534cb37b784078edf66b17d

File: DOCUMENTATION.md (hunk 1 above)
Comment (tombye) [reply DOCUMENTATION.md]: I think this has come back in, maybe with the other changes.

**Pull request merged**